lib/Doctrine/DBAL/Schema/SqliteSchemaManager.php

```diff
@@ -116,8 +117,11 @@ public function listTableForeignKeys($table, $database = null)
         $tableForeignKeys = $this->_conn->fetchAll($sql);
 
         if ( ! empty($tableForeignKeys)) {
-            $createSql = $this->_conn->fetchAll("SELECT sql FROM (SELECT * FROM sqlite_master UNION ALL SELECT * FROM sqlite_temp_master) WHERE type = 'table' AND name = '$table'");
-            $createSql = isset($createSql[0]['sql']) ? $createSql[0]['sql'] : '';
+            $createSql = $this->_conn->fetchColumn(
+                'SELECT sql FROM (SELECT * FROM sqlite_master UNION ALL SELECT * FROM sqlite_temp_master) WHERE type = "table" AND name = ?',
+                array(str_replace('.', '__', $table))
+            );
```

Comment:
@hason this change is not part of your PR, is it? Can we try to not mix different things here?